src/components/views/voip/VideoLobby.tsx

```diff
@@ -169,17 +162,23 @@ const VideoLobby: FC<{ room: Room }> = ({ room }) => {
 
     const connect = async () => {
         setConnecting(true);
+
+        const audioOutputId = MediaDeviceHandler.getAudioOutput();
+        const audioOutput = audioOutputs.find(d => d.deviceId === audioOutputId) ?? audioOutputs[0];
```

Comment:
I wonder if we should add this as a method to `MediaDeviceHandler`, what do you think?

Reply:
This line has been simplified now that I'm setting the default devices through `MediaDeviceHandler`, and `audioOutputs` has to come from within this function anyways rather than `MediaDeviceHandler` to avoid having to request it from the browser again, so I'd argue that this is fine now